coreutils/Makefile

```diff
@@ -300,10 +300,10 @@ ${COREUTILS_NOLINK_DIR}:
 	# Generate line coverage information
 	if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
 		${CURDIR}/../utils/coreutilscov.sh "$*" "$$OUTPUT_DIR" ${COREUTILS_COV_MAX_TIME}
```

Comment:
@rasoolmaghareh This line has to end with ; \
This is to continue in the same subshell with the next line, as we still need to close the if with the fi.